py-polars/docs/source/reference/series/computation.rst

```diff
@@ -19,6 +19,7 @@ Computation
     Series.cos
     Series.cosh
     Series.cot
+    Series.count
```

Comment:
would `aggregation` be a better section, as that's where it is for `Expr`?

Reply:
Thanks for the question and the quick review. Yes, I do think that's a better section - thanks for that.

It is in both `aggregation` and `functions` for `Expr`, so initially that was why I moved it into `computation`. Upon reflection `aggregation` is indeed much better suited. Will update now.